tests/data.json

```diff
@@ -218,6 +225,19 @@
                 {"data_type": "read"}
             ]
         },
+        "/read-groups": {
+            "valid": {
+                "paired_end": 1,
+                "lane_number": 3,
+                "flow_cell": "FL123",
+                "quality_scale": "Illumina15"
+            },
+            "invalid": [
```

Comment:
Maybe good to include negative number values here too for the attrs that need to be positive numbers